app/src/main/java/com/infomaniak/swisstransfer/ui/utils/HumanReadableSizeUtils.kt

```diff
@@ -24,10 +24,20 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.pluralStringResource
 import com.infomaniak.swisstransfer.R
+import com.infomaniak.swisstransfer.ui.components.FileUi
 
 object HumanReadableSizeUtils {
-    fun getHumanReadableSize(context: Context, sizeInBytes: Long): String {
-        return Formatter.formatShortFileSize(context, sizeInBytes)
+    private const val TOTAL_FILE_SIZE: Long = 50_000_000_000L
+
+    private fun getHumanReadableSize(context: Context, sizeInBytes: Long) = Formatter.formatFileSize(context, sizeInBytes)
```

Comment:
Why did the format change from formatShortFileSize to formatFileSize?

Reply:
I asked the UI/UX, he wanted the same logic as the web version where there's only 1 decimal number and if the decimal is 0 as in "50.0 GB" the decimal would be gone. "50.00 GB" felt kind of cluttered compared to "50 GB" for him.

I don't know what's the closest we can get to what he wants?